### ISSUE TITLE
Unable to see output through Deeplens inspite of successful deployment to device

### ISSUE THREAD

**Issue by duttasantanuGH** [mahendrabairagi/DeeplensWorkshop#9]
Hi
I am trying to implement "Build your own object classification model in SageMaker and import it to DeepLens" as per link https://aws.amazon.com/blogs/machine-learning/build-your-own-object-classification-model-in-sagemaker-and-import-it-to-deeplens/ 
I have ensured that the mxnet==0.12.1 is updated. My training, deployment is successful to the device but I am not being able to see output. I have used the following commnad in Deeplens terminal window as per guidelines:
mplayer -demuxer lavf -lavfdopts format=mjpeg:probesize=32 /tmp/results.mjpeg

For other deployed project, I could able to see output successfully. 

Please help me in resolving the issue.

aws_cam@Deepcam:~$ mplayer -demuxer lavf -lavfdopts format=mjpeg:probesize=32 /tmp/results.mjpeg
MPlayer 1.2.1 (Debian), built with gcc-5.3.1 (C) 2000-2016 MPlayer Team
mplayer: could not connect to socket
mplayer: No such file or directory
Failed to open LIRC support. You will not be able to use your remote control.

Playing /tmp/results.mjpeg.
File not found: '/tmp/results.mjpeg'
Failed to open /tmp/results.mjpeg.

**Comment by yagnesh-tendulkar** [comment]
@duttasantanuGH are you able to solve this issue? I am also facing the same issue. If you solve that issue let me know what i have to do.

**Comment by Balsaraswathi-S** [comment]
any solution for this issue? if yes kindy do rply

**Comment by yagnesh-tendulkar** [comment]
Yeah i just update the AWS deeplens and then i am able to run my code successfully. Just make sure that the software is up-to-date and also check with the lambda package.
Hope it helps. 
Thanks and regards.

**Comment by Balsaraswathi-S** [comment]
hey yagnesh can u pls tell me how to update the deeplens
i am solving this issue for two days..

**Comment by Balsaraswathi-S** [comment]
the deployment of project is done successfully...while streaming the deeplens i am getting this error..

**Comment by Balsaraswathi-S** [comment]
i update the deeplens and i still getting the same error

**Comment by Balsaraswathi-S** [comment]
can u pls tell the solution to solve this error

**Comment by yagnesh-tendulkar** [comment]
Actually i am facing the issues while updating the software so i go with factory reset. i.e. reinstall complete deeplens package and OS in deeplens then it works for me.

**Comment by yagnesh-tendulkar** [comment]
https://docs.aws.amazon.com/deeplens/latest/dg/deeplens-device-factory-reset-preparation.html.
Follow the docs so that you may get to how do reset the device.

**Comment by Balsaraswathi-S** [comment]
okiee bro..thanks for the help.. ill ping once i got op..
